manuscript/the-meat-start.txt

```diff
@@ -86,14 +84,14 @@ We're going to follow this simple approach:
 * React owns the DOM
 * d3 calculates properties
 
-This way we leverage both React and d3.js for what they’re best at.
+This way we leverage both React and d3.js for their best functions.
```

Comment:
"This way we leverage both React and d3.js for their best functions and not much else."

-What about that?